integrations/extensions/README.md

```diff
@@ -1,11 +1,79 @@
-# Watson Assistant extension starter kits
+# Watson Assistant Extensions Starter Kit
+
+## Introducing Extensions :tada:
 
 These starter kits provide everything you need to create assistants that use custom extensions to access external services using REST APIs.
 
-Each starter kit includes the following:
+Each starter kit includes:
+
+- A valid OpenAPI 3.0 specification describing the REST API of a third-party service. These OpenAPI documents have been tested and validated with Watson Assistant.
+- An actions skill JSON file that uses the extension created from the OpenAPI spec. You can import this file into an assistant and then edit these actions to access the extension you create.
+- A README containing API specific instructions on how to set up the extension.
+
+### How to Use This Repository
+This guide is for using a **starter kit** for your custom extension. For our full docs on how to create and customize extensions, visit [Building a Custom Extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension). 
+
+We're providing and maintaining **two** starter kits to be used as tutorials: `HubSpot` and `MetaWeather`. The **basic** path will help you get your first custom extension started. Then, you can move on to the **advanced** path for more complex OpenAPI specs and actions skills that showcase the full capabilities of extensions, and can use as a reference to build your own specs and skills.
```

Comment:
why only two?

Reply:
I talked about this with @mitchmason .. actively maintaining an advanced and basic spec for all the extensions might become nightmare-ish, since the APIs are all different for the extensions. Hopefully the idea is that contributors will step in to actively maintain any future broken specs and create issues for it... but we will make sure that HubSpot and MetaWeather are always working and usable